library/aws/policies.ts

```diff
@@ -1,409 +1,132 @@
-import * as TF from '../../core/core.ts';
 import * as AR from '../../providers/aws/resources.ts';
 import { ArnSecret } from './secrets.ts';
+import {assumeServiceRole, autoscalingGroupEnableSetInstanceProtection as autoscalingGroupEnableSetInstanceProtectionStatements, ecrModifyAll, ecrReadonly, edModify, firehosePutRecord, lambdaInVpc, NamedPolicy, newNamedPolicy, publishMetrics, putLogs, putAnyLogs, route53ModifyZone, s3ModifyBuckets, s3PublicReadonly, s3ReadonlyBuckets, secretReadOnly, sesAllActions, snsFullAccess, snsPostSms, sqsQueueModify, Statement} from './policies_v2.ts';
+export type { NamedPolicy, Statement } from './policies_v2.ts';
 
-export interface NamedPolicy {
-  name: string;
-  policy: {};
-}
+/** 
```

Comment:
Is this comment relevant?

Reply:
It's intended as guidance of the file being deprecated